ant/src/org/jetbrains/kotlin/ant/Kotlin2JvmTask.kt

```diff
@@ -94,8 +93,8 @@ class Kotlin2JvmTask : KotlinCompilerBaseTask() {
         val javaBin = javaHome + separator + "bin" + separator + "java"
         val redirector = Redirector(this)
 
-        val msgRendererPropKey = MessageRenderer.PROPERTY_KEY
-        val msgRendererPropVal = MessageRenderer.PLAIN_FULL_PATHS.name
+        val msgRendererPropKey = "org.jetbrains.kotlin.cliMessageRenderer"
```

Comment:
Please inline these variables

Reply:
I'm afraid I don't quite understand what you mean, should I change it from 
```
val msgRendererPropKey = "org.jetbrains.kotlin.cliMessageRenderer"
val msgRendererPropVal = "FullPath"
command.add("-D$msgRendererPropKey=$msgRendererPropVal")
```
to
```
command.add("-Dorg.jetbrains.kotlin.cliMessageRenderer=FullPath")
```
?

Reply:
Yes, exactly

Reply:
(I'm saying "inline" because this is how the corresponding action is called in IntelliJ, see the menu -> Refactor -> _Inline_ property)

Reply:
Understood, thanks.